server/templates/diff.html

```diff
@@ -97,3 +97,36 @@
     </table>
   </div>
 {% endmacro %}
+
+{% macro diff_lines(file) %}
+  <div class="source-file table-responsive"
+    data-filename={{ filename }}>
+    <table class="highlight">
+      <tbody>
+        {% for line in file.lines %}
```

Comment:
This looks copy-pasted from above, but without the `<thead>`. The part starting from `{% for line in file.lines %}` should be factored into a jinja function